src/App.js

```diff
@@ -28,16 +31,55 @@ class App extends React.Component {
           {
             id: 5,
             name: 'Crime',
-          }]
+          }],
+      movies: [
+          {
+              imageUrl: 'https://www.movieposters.com/cdn/shop/products/b892c2f862023362da3e66ec2b92a699_90de31ac-e4ca-476e-8cc0-f634509f364b_480x.progressive.jpg?v=1573585334',
+              name: 'Scarface',
+              releaseYear: 1983,
+              relevantGenres: [
+                  'Action',
+                  'Drama',
+              ],
+              rating: 8.3,
+              duration: 170,
+              description: 'In 1980 Miami, a determined Cuban immigrant takes over a drug cartel and succumbs to greed.',
+          },
+          {
+              imageUrl: 'https://www.movieposters.com/cdn/shop/files/Casino.mpw.102809_480x.progressive.jpg?v=1707421876',
+              name: 'Casino',
+              releaseYear: 1995,
+              relevantGenres: [
+                  'Crime',
+                  'Thriller',
+              ],
+              rating: 8.2,
+              duration: 178,
+              description: 'In Las Vegas, two best friends - a casino executive and a mafia enforcer - compete for a gambling empire and a fast-living, fast-loving socialite.',
+          }
+      ],
+      selectedMovie: null,
+      sortBy: '',
+      sortByOptions: [
```

Comment:
If you don't change the sortByOptions value you don't need to store it in state.